layout/_partials/head/head.swig

```diff
@@ -107,6 +107,8 @@
     scheme: '{{ theme.scheme }}',
     version: '{{ version }}',
     sidebar: {{ theme.sidebar | json_encode }},
+    back2top: {{ theme.back2top.enable }},
```

Comment:
Need to refactor this under json, no need both `back2top` and `back2top_sidebar` parameters.

Reply:
Yes, it's a good idea. You can do it.